src/Main/showToast.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import SafeHTMLMessage from '@folio/react-intl-safe-html';
+
+// eslint-disable-next-line import/prefer-default-export
+export function showToast(messageId, messageType = 'success', values = {}) {
```

Comment:
Do you think it's the best way - to refuse common function and implement the local one? Why you can't just pass `<SafeHTMLMessage id={messageId} values={values} />` as `message` to the common one?